integration-tests/tests/pool/2.liquidator.spec.ts

```diff
@@ -159,35 +159,88 @@ describe("LendingPool: Liquidation (receive underlying assets)", function () {
     });
 
     // TODO: requires optimization
-    it("Case 5: Liquidator liquidates Borrower's positions", async function () {
+    it("Case 5: Liquidator liquidates Borrower's position partialy", async function () {
```

Comment:
Does this test still fails due to budget limits? If not, could you please remove the "TODO" above?